### ISSUE TITLE
Failed to install

### ISSUE THREAD

**Issue by zhouhao27** [niutech/node.php#13]
Got the following messages:

> Installing Node.js:
> Failed. Error: 
> Try putting node folder via (S)FTP, so that >/home/users/htdocs/node/bin/node exists.

What could be the reason? Thanks.

**Comment by niutech** [comment]
This means one of these chained commands did not succeed:

```
tar -xzf NODE_FILE 2>&1 &&
mv node-NODE_VER-linux-NODE_ARCH NODE_DIR &&
touch nodepid &&
rm -f NODE_FILE
```

Connect using SSH or (S)FTP and check if the file `node-*-linux-*.tar.gz`, the folder `node` and the file `nodepid` exist. If not, manually extract the file `node-*-linux-*.tar.gz` to the folder `node` and create the empty file `nodepid`.

**Comment by gaberilde** [comment]
its tar.xz not tar.gz now

**Comment by darksight149** [comment]
Is it issue already fixed? I have also the same issue

**Comment by niutech** [comment]
No, there still exist https://nodejs.org/dist/v9.1.0/node-v9.1.0-linux-x86.tar.gz and https://nodejs.org/dist/v9.1.0/node-v9.1.0-linux-x64.tar.gz
